Update schedule.html

story645 opened the pull request:
Same as workshop but now on the right page-condensed the info/dropped redundancies.

kaymmm commented:
don't modify this repository directly. these files are generated programmatically. please make any changes as pull requests in the gcdrb repository.

story645 committed:
Update schedule.html

Same as workshop but now on the right page-condensed the info/dropped redundancies.
